### PR TITLE
Add --no-plot mode to speed up postprocess

### PR DESCRIPTION
**Pull request by colindaven**

### CODE REVIEW

File: wochenende_postprocess.sh [hunk 1]
```diff
@@ -82,17 +87,24 @@ echo "INFO: Completed Sambamba depth and filtering"
...
-cp ../*_window.txt.filt.csv .
-bash runbatch_wochenende_plot.sh >/dev/null 2>&1
-#wait
-echo "INFO: Sleeping for " $sleeptimer
-sleep $sleeptimer
-cd $bamDir
-echo "INFO: Completed Wochenende plot"
+if [ $1 = "--no-plot" ] 
+then
+    echo "INFO: Found --no-plot argument: Plot mode disabled"
```
Comment (B1T0) [inline wochenende_postprocess.sh:92]: Why do we state this twice?

**Review by B1T0 — APPROVED** [review]
Looks good to me :)

File: wochenende_postprocess.sh (hunk 1 above)
Comment (colindaven) [reply wochenende_postprocess.sh:92]: cheers for looking Tobias.
If you're just referring to the code snippet you have, if I'm correct, line 90 does the if statement but produces no on screen output, while 92 prints the logging statement INFO for the user.
For me, that's not a problem. Or do you mean another section ?

Comment (B1T0) [reply wochenende_postprocess.sh:92]: It seems to be a duplicate of line 35

**Comment by colindaven** [comment]
I think it's ok, a lot of info is given out at the start, so it makes sense to report the plot mode is skipped when the program gets to the actual section. However, it's not really so important.

**Pull request merged**